dev_container

```diff
@@ -41,15 +41,13 @@ init_globals() {
     HOLOHUB_CONTAINER_BASE_NAME=holohub
     export HOLOHUB_CONTAINER_BASE_NAME
 
-
-    DO_DRY_RUN="false"  # print commands but do not execute them. Used by run_command
+    DO_DRY_RUN="false" # print commands but do not execute them. Used by run_command
```

Comment:
Could you please provide the formatting rules that you used? We don't currently have a HoloHub formatting standard, it would be good to standardize around one set of rules. The reformatting looks fine, but want to guard against format changes each time someone new contributes with a different IDE.

Reply:
I used [foxundermoon.shell-format](https://marketplace.visualstudio.com/items?itemName=foxundermoon.shell-format) VS code extension which is based on https://github.com/mvdan/sh#shfmt